spec/models/item_spec.rb

```diff
@@ -45,13 +45,6 @@
     expect(item).to be_invalid
   end
 
-  it "won't create an item without an image_url" do
```

Comment:
Well Items dont need a image url anymore as we set a default if no url is provided see #99